kubernetes-model-generator/kubernetes-model-common/src/main/java/io/fabric8/kubernetes/model/annotation/Categories.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2024 Ericsson Software Technology
```

Comment:
Hi @MichaelMorrisEst 
The header/license files cannot be changed.
Is it OK if you fallback to the project template?
If not, I'm afraid we won't be able to merge this PR and discard its content.

https://github.com/fabric8io/kubernetes-client/issues/2045

Reply:
The company name. Style checks and license checks won't pass unless the header text is identical to the rest.

In case I need to make changes to the template or any of its variables I would need to discuss with Red Hat legal first.
In any case, at most the `{owner}` name could be changed to something like `The Fabric8 Kubernetes Client authors` (after their approval).

Reply:
I've updated as suggested to allow the PR to proceed, but I do not think this is in keeping with the norms for Apache 2.0 licensed software. Id suggest you should check with Red Hat legal what company name they expect in the case of contributions from companies other than Red Hat. I believe the norm here is the company name of the contributor.